### PR TITLE
Use the name "Thieves' Hideout" wherever applicable, and rename "Composer's Grave" to "Royal Family's Tomb"

### PR DESCRIPTION
**Pull request by puddydoo**
Based on #1108 but does not add any features, just makes the randomizer refer to "Thieves' Hideout" wherever applicable rather than lumping it in with Gerudo's Fortress.

### CODE REVIEW

**Comment by ETR-BTF** [comment]
By that logic Dampe's grave & Windmill should be "?" and GTG should be "Gerudo Training Ground" (without the "s"). r0bd0g'll be happy about that. ;)

**Comment by r0bd0g** [comment]
I mean I think the names should match where reasonable. This one's probably reasonable...
Did you get the dungeon info screen? That seemed like the hard part.

**Comment by puddydoo** [comment]
I was going to rename GTG but I thought renaming an entire dungeon is a big change that might have to be a separate PR.

**Comment by puddydoo** [comment]
Note that the Gerudo guard key locations are color-coded, this is because each cell has a Gerudo banner over it with a different color. For hideout ER (though it's not in this PR) the position of the rooms can't really be used to distinguish them. But I'm also somewhat unsure of whether to use the colors alone because color-blind players might not be able to distinguish that either.

**Comment by r0bd0g** [comment]
Composer's Grave->Royal Family's Tomb is another good catch IMO.

**Comment by puddydoo** [comment]
Renamed the guard locations again because I noticed that the number of torches in their rooms was different. This way there will be no confusion from ER or colorblindness, and also it can carry over to OoT 3DS where the banner in the 3-torch room is recolored from green to gray and the one in the 4-torch room is recolored from olive to green.

File: EntranceShuffle.py [hunk 1]
```diff
@@ -255,8 +255,8 @@ def build_one_way_targets(world, types_to_include, exclude=(), target_region_nam
                         ('Graveyard Shield Grave -> Graveyard',                             { 'index': 0x035D })),
     ('Grave',           ('Graveyard -> Graveyard Heart Piece Grave',                        { 'index': 0x031C }),
                         ('Graveyard Heart Piece Grave -> Graveyard',                        { 'index': 0x0361 })),
-    ('Grave',           ('Graveyard -> Graveyard Composers Grave',                          { 'index': 0x002D }),
-                        ('Graveyard Composers Grave -> Graveyard',                          { 'index': 0x050B })),
+    ('Grave',           ('Graveyard -> Royal Familys Tomb',                                 { 'index': 0x002D }),
```
Comment (mzxrules) [inline EntranceShuffle.py:258]:
```suggestion
    ('Grave',           ("Graveyard -> Royal Family's Tomb",                                 { 'index': 0x002D }),
```
repeat in other spots

Comment (puddydoo) [reply EntranceShuffle.py:258]: Other regions don't have apostrophes in the name

Comment (Zannick) [reply EntranceShuffle.py:258]: This also needs to match the actual region name.

**Comment by Zannick** [comment]
I think we also ought to rename the `gerudofortress` setting and the `can_finish_gerudofortress` helper rule since they exclusively deal with the carpenter rescue quest in the Thieves' Hideout.

**Comment by r0bd0g** [comment]
I'm gonna make a suggestion - maybe this isn't the right place for it - but maybe going through the kitchen with no items, and the equivalent glitchless jump outside in fortress should be two separate tricks? I'm always annoyed I can't enable the current trick only b/c going through kitchen (potentially as child) is so much more difficult than the jump outside.

**Comment by Zannick** [comment]
Let's deal with that separately.

**Review by Zannick — APPROVED** [review]
> I think we also ought to rename the `gerudofortress` setting and the `can_finish_gerudofortress` helper rule since they exclusively deal with the carpenter rescue quest in the Thieves' Hideout.

I'm going to attempt this after I merge this and double check that everything was successfully renamed, if you're not in the process of doing it now.

**Pull request merged**